iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/content/WorkflowTest.java

```diff
@@ -13,10 +13,18 @@
 public class WorkflowTest extends AbstractOozieWorkflowTestCase {
 
     @Test
-    public void testImportContentUrlWorkflow() {
+    public void testImportContentUrlWorkflowWithObjectStoreAsBackend() {
         OozieWorkflowTestConfiguration wfConf = new OozieWorkflowTestConfiguration();
         wfConf.setTimeoutInSeconds(720);
-        testWorkflow("eu/dnetlib/iis/wf/importer/content_url/chain/sampletest", wfConf);
+        testWorkflow("eu/dnetlib/iis/wf/importer/content_url/chain/objectstore_based_importer", wfConf);
+    }
+    
+    //@Test
+    //TODO reenable this test once Hive is installed on CI Test hadoop cluster
```

Comment:
A perfect case for [`@Disabled("TODO reenable this test once Hive is installed on CI Test hadoop cluster")`](https://junit.org/junit5/docs/current/user-guide/#writing-tests-disabling), I think.

Even [`@DisabledIf`](https://junit.org/junit5/docs/current/user-guide/#writing-tests-conditional-execution-custom) could be fun if Hive is easily detectable, but that's not necessarily a good idea.

Reply:
I will stick to `@Disabled` instead of `@DisabledIf` because in the end we want to have it enabled just need to fix the environment.